# This message consists of anything that needs updating
# for each sensor type
bool[] sensor_type_is_actives
# for each sensor
uint16[] sensor_responses
